src/java/org/apache/cassandra/tools/StandaloneScrubber.java

```diff
@@ -35,75 +35,75 @@
 import org.apache.cassandra.service.CassandraDaemon;
 import org.apache.cassandra.utils.OutputHandler;
 
+import java.util.concurrent.Executor;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ThreadPoolExecutor;
+
 import static org.apache.cassandra.tools.BulkLoader.CmdLineOptions;
 
-public class StandaloneScrubber
-{
```

Comment:
Looks like cass  project uses curly braces on new line,  it might make sense to make sense to setup our formatter to match? otherwise these PRs  might be quite noise with formatting changes.